indico/modules/events/client/js/display.js

```diff
@@ -26,10 +31,21 @@ import './favorite';
       });
     }
 
-    $(document).on('click', '[data-note-editor]', function(evt) {
-      evt.preventDefault();
-      openAjaxDialog($(this));
-    });
+    const containers = document.querySelectorAll('.edit-minutes-container');
+    if (containers) {
+      containers.forEach(container => {
+        ReactDOM.render(
+          React.createElement(NoteEditor, {
+            icon: container.dataset.icon !== undefined,
+            title: container.dataset.title,
+            objectType: container.dataset.itemType,
```

Comment:
No longer used? (I think you already removed it from the dataset, so it's always `undefined` here)